staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go

```diff
@@ -797,17 +798,6 @@ func (vs *VSphere) InstanceShutdownByProviderID(ctx context.Context, providerID
 	return !isActive, nil
 }
 
-// InstanceMetadataByProviderID returns metadata of the specified instance.
-func (vs *VSphere) InstanceMetadataByProviderID(ctx context.Context, providerID string) (*cloudprovider.InstanceMetadata, error) {
```

Comment:
Safe to remove since InstancesV2 is disabled:

https://github.com/kubernetes/kubernetes/blob/d3c1e81d5e6796db9389cd73073f48d6a37c4621/staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go#L563-L566